### ISSUE TITLE
Download not working when adding range header

### ISSUE THREAD

**Issue by mayankagarwal09** [lingochamp/okdownload#301]
### Problem Describe

When adding Range header to task download is not working.
Maybe you are checking for response 200 before starting download and range header returns 206

**Comment by rantianhua** [comment]
I don't understand your problem clearly. OkDownload uses range to support downloading from breakpoint and handles 200 & 206 correctly. So, what you mean adding range header? Actually, range header is handled by OkDownload automatically. 

Maybe you mean that you want to use range to download partial file?

**Comment by mayankagarwal09** [comment]
Yes i want to use range header to partially download the file but is downloading the whole file

**Comment by rantianhua** [comment]
Yes, OkDownload alway download the whole file. Could you tell me your use case? And why you just  want to download partially file? This may be a new feature for OkDownload.

**Comment by mayankagarwal09** [comment]
I am working on an app where different parts of the file are downloaded by different users.
I need to download a file partially only.
I think it should not be difficult to add support for it as you can just forward the request along with the headers added and save the response as received
